components/Form.js

```diff
@@ -0,0 +1,218 @@
+import { useState } from 'react';
+import { nanoid } from 'nanoid';
+import { useRouter } from 'next/router';
+import { toast } from 'react-toastify';
+import Icon from '@mdi/react';
+import { mdiSend } from '@mdi/js';
+import {
+  FormControl,
+  TextField,
+  Select,
+  MenuItem,
+  TextareaAutosize,
+  FormLabel,
+  Radio,
+  RadioGroup,
+  FormControlLabel,
+  Stack,
+  Box,
+  Button,
+} from '@mui/material';
+
+import styled from 'styled-components';
+
+import { today } from '../utils/date';
+
+const StyledTextareaAutosize = styled(TextareaAutosize)`
+  padding: 8px;
+  margin-bottom: 5px;
+`;
+
+export default function Form() {
+  const router = useRouter();
+  const [form, setForm] = useState({
+    title: '',
+    location: '',
+    activity: '',
+    ageGroup: '',
+    description: '',
+    date: '',
```

Comment:
```suggestion
    date: new Date(),
```

without form validation this could turn out to be invalid on the meeting page. As a default, you should set new Date(). This _might_ interfere with your date picker element, so you might have to rework my suggestion, but don't keep the default value as an empty string
<img width="226" alt="image" src="https://user-images.githubusercontent.com/15978350/174192602-f37cac95-0186-4207-921c-5786ba3937d4.png">

Reply:
yes, my form validation is still missing